docker-compose.yaml

```
@@ -124,13 +124,13 @@ services:
       IMP_SPARK_MASTER_URL: spark://spark-master:7077
       IMP_SPARK_DRIVER_HOST: test-container
       IMP_MINIO_IMPORTER_RAW_BUCKET: cts-output
+      IMP_SQL_WAREHOUSE_PREFIX: s3a://cdm-lake/user_warehouse/
```

```suggestion
      IMP_SQL_WAREHOUSE_PREFIX: s3a://cdm-lake/users-sql-warehouse
```
This is what we should use on the server, but it’s fine to use this locally.

---

yeah, I figured for testing purposes it doesn't matter